network/controller/bootstrap/network_bootstrap.go

```diff
@@ -37,19 +37,29 @@ type NetworkBootstrapper struct {
 	authController      *AuthorizationController
 	challengeController *ChallengeResponseController
 	nodeKeeper          network.NodeKeeper
+	firstPulseTS        int64
```

Comment:
firstPulseTS is never initialized to correct timestamp

Reply:
fixed